intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -109,41 +109,43 @@ \section{First Time connection to the \hpc}
       \keys{Open}.
   \end{enumerate}
 
-%ENDIF windos
+%ENDIF windows
 \fi
 
 \ifmacORlinux
-\subsection{Connect}
-\label{sec:connect}
 
-The \hpc is only accessible from within the \university network, but you can
-get external access (e.g., from home) by using a VPN connection.
+  \subsection{Connect}
+  \label{sec:connect}
 
-Open up a terminal and enter the following command to connect to the \hpc.
-\ifmac
-You can open a terminal by navigation to Applications and then Utilities in the finder and open Terminal.app, or enter Terminal in Spotlight Search.
-\fi
-
-\begin{prompt}
-%\shellcmd{ssh \userid{}@\loginnode{}}%
-\end{prompt}
+  The \hpc is only accessible from within the \university network, but you can
+  get external access (e.g., from home) by using a VPN connection.
 
-Here, user \userid wants to make a connection to the ``\hpcname'' cluster at
-\university via the login node ``\loginnode'', so replace \userid with your own
-userid in the above command.
-
-The first time you make a connection to the login node, you will be asked to
-verify the authenticity of the login node, e.g.,
+  Open up a terminal and enter the following command to connect to the \hpc.
+  \ifmac
+  You can open a terminal by navigation to Applications and then Utilities in the finder and open Terminal.app, or enter Terminal in Spotlight Search.
+  \fi
 
-\begin{prompt}
-%\shellcmd{ssh \userid{}@\loginnode{}}%
-The authenticity of host %\loginnode% (<IP-adress>)
-can't be established.
-RSA key fingerprint is %\loginfingerprint%
-Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
-\end{prompt}
+  \begin{prompt}
+  %\shellcmd{ssh \userid{}@\loginnode{}}%
+  \end{prompt}
 
+  Here, user \userid wants to make a connection to the ``\hpcname'' cluster at
+  \university via the login node ``\loginnode'', so replace \userid with your own
+  userid in the above command.
+  
+  The first time you make a connection to the login node, you will be asked to
+  verify the authenticity of the login node, e.g.,
+  
+  \begin{prompt}
+  %\shellcmd{ssh \userid{}@\loginnode{}}%
+  The authenticity of host %\loginnode% (<IP-adress>)
+  can't be established.
+  RSA key fingerprint is %\loginfingerprint%
+  Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
+  \end{prompt}
```

Comment:
I don't think we have those available. Also, we will then need to keep that info up-to-date.
Any suggestions?